earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -33,110 +41,139 @@ sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
 
-The GEE build scripts will detect this compiler automatically. However, if you wish switch the environment to use GCC 4.8 in a shell, run: 
-```source /opt/rh/devtoolset-2/enable```
+The GEE build scripts will detect this compiler automatically. However, if you
+wish switch the environment to use GCC 4.8 in a shell, execute:
 
+```bash
+source /opt/rh/devtoolset-2/enable
+```
 
 ## Install additional packages
-_Note: if you are upgrading from GEE 5.1.3 or earlier, you must run this step __after__ uninstalling older versions of GEE.  Otherwise, some of the prerequisites will be missing and your build will fail._
+
+__NOTE:__ If you are upgrading from GEE 5.1.3 or earlier, you must run this step
+_after_ uninstalling older versions of GEE. Otherwise, some of the
+prerequisites will be missing and the build will fail.
 
 ### CentOS 7 and RHEL 7
 Execute: 
 ```
-sudo yum install -y scons perl-Perl4-CoreLibs xorg-x11-server-devel python-devel perl-Alien-Packages  \
-    openssl-devel libxml2-devel libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel   \
-    libmng-devel libcap-devel libpng12-devel libXmu-devel freeglut-devel zlib-devel libX11-devel bison-devel  \
-    openjpeg-devel openjpeg2-devel geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel cmake \
-    rpm-build rsync
+sudo yum install -y scons perl-Perl4-CoreLibs xorg-x11-server-devel \
+  python-devel perl-Alien-Packages openssl-devel libxml2-devel \
+  libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel \
+  libmng-devel libcap-devel libpng12-devel libXmu-devel freeglut-devel \
+  zlib-devel libX11-devel bison-devel openjpeg-devel openjpeg2-devel \
+  geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel \
+  cmake rpm-build rsync
 ```
+
 ### CentOS 6 and RHEL 6
-Execute: 
-```
-sudo yum install -y scons perl-Perl4-CoreLibs xorg-x11-server-devel python-devel perl-Alien-Packages  \
-    openssl-devel libxml2-devel libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel   \
-    libmng-devel libcap-devel libpng-devel libXmu-devel freeglut-devel zlib-devel libX11-devel bison-devel  \
-    openjpeg-devel openjpeg2-devel geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel cmake
-    rpm-build rsync shunit2
-```
-If you get an error about git dependency conflicts, consider experimenting with the `--skip-broken` parameter.
+Execute:
 
+```bash
+sudo yum install -y scons perl-Perl4-CoreLibs xorg-x11-server-devel \
+  python-devel perl-Alien-Packages openssl-devel libxml2-devel \
+  libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel \
+  libmng-devel libcap-devel libpng-devel libXmu-devel freeglut-devel \
+  zlib-devel libX11-devel bison-devel openjpeg-devel openjpeg2-devel \
+  geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel \
+  cmake rpm-build rsync shunit2
+```
 
+If you encounter an error about git dependency conflicts, consider 
```

Comment:
Minor: You don't have to fix, since it's probably not from your commits.

This is rather unclear.  We should either document the error message and how to fix it, or remove this.  Developers can experiment all kinds of ways, and probably don't need us to tell them how to do that.